mocks/commitexporter_github.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
how we create/edit this files?

Reply:
Very carefully.

Just kidding! We edit it with [Mockoon](https://mockoon.com/).

We should probably document how we use mockoon for new tests. Here's what we do:

We set it up as an intercepting proxy, and record responses from openshift, github, or other API providers. We can then reuse those responses for mocked integration tests.

Using the GUI is good for manual testing, and is how we set up new data. But for automated testing, you can run it from the CLI. That's what our script does.